pkg/machines/actions.es6

```diff
@@ -83,9 +91,16 @@ export function vmDesktopConsole(vm, consoleDetail) {
  * and scheduled on later.
  *
  * @param action I.e. getAllVms()
+ * @param timeout Non-default timeout
+ * @param name VM name (if nonexisting or usage not enabled, action will not be run);
+ *             if not given, use global timeout
+ * @param connectionName Connection for #name
  */
-export function delayPolling(action, timeout) {
+export function delayPolling(action, timeout, name, connectionName) {
     return (dispatch, getState) => {
+        if (name && !usagePollingEnabled(getState(), name, connectionName))
```

Comment:
Indeed I would have preferred it in `doUsagePolling()`, but due to how we constructed the provider API the implementation in libvirt.es6 does not have access to the redux state.

Good point about passing on `name` and `connectionName`. I'm reasonably sure that it's only a theoretical case as this is only being invoked if the "Usage" tab is visible (which it isn't while cockpit is hidden). However, there is no harm in passing it on indeed, and it looks "more correct". Fixed.